common/src/main/scala/explore/GraphQLSchemas.scala

```diff
@@ -0,0 +1,25 @@
+package explore
+
+import clue.macros.GraphQLSchema
+import explore.model.SiderealTarget
+import explore.model.enum._
+
+object GraphQLSchemas {
+  @GraphQLSchema(debug = false)
+  object ObservationDB {
+    object Scalars {
+      type Cloudcover       = CloudCover
+      type Imagequality     = ImageQuality
+      type Skybackground    = SkyBackground
+      type Watervapor       = WaterVapor
+      type Obsstatus        = ObsStatus
+      type Targetobjecttype = TargetType
```

Comment:
Yes. The macro actually converts from `snake_case` to `CamelCase`.

Since the types in this particular schema (from Hasura) are just lowercase, they get just capitalized (the ones on the left). The ones on the right are our `Enumerated` types.

If the types in Hasure were snake cased (eg: `sky_background`), then we would get the same resulting casing and we would have to qualify our enums (eg: `enum.SkyBackground`).